test/metabase/test/util.clj

```diff
@@ -33,7 +33,6 @@
    [metabase.models.setting :as setting]
    [metabase.models.setting.cache :as setting.cache]
    [metabase.models.timeline-event :as timeline-event]
-   [metabase.notification.test-util :as notification.tu]
```

Comment:
need to remove this so `metabase.notification.test-util` can import `metabase.test` without circular dependency